test/system/src/main/java/io/pravega/test/system/framework/kubernetes/K8sClient.java

```diff
@@ -196,6 +196,11 @@ public V1Namespace createNamespace(final String namespace) {
     @SneakyThrows(ApiException.class)
     public CompletableFuture<List<V1PodStatus>> getStatusOfPodWithLabel(final String namespace, final String labelName, final String labelValue) {
         CoreV1Api api = new CoreV1Api();
+
+        // Workaround based on okhttp issue https://github.com/square/okhttp/issues/2300
```

Comment:
Could you add issue #3361 to this comment, pls?

Reply:
done.